mkdocs.yml

```diff
@@ -16,7 +16,7 @@ nav:
     - '(Working Draft) JSON Schema': https://opendi-org.github.io/json-schema
```

Comment:
Should this point to opendi.org or GitHub.io?

Reply:
In testing, I found that these end up resolving to their corresponding `opendi.org` URLs anyway, so I haven't been strict about replacing them yet, especially since we're still awaiting certs for a couple of them.

Went ahead and changed this to the `http` opendi.org ones for now anyway, since that's at least more consistent than this. See c0cb888